salesagility/SuiteCRM
# Update email compose from dropdown

## Description




- Allow email compose to send using outbound-only accounts
- Update email compose from dropdown
- Only display outbound and system accounts on email compose from
- Remove option to allow user to send as themselves from configuration
- Add "Enable Legacy Email Compose Behaviour" to the Email Settings screen to:
     - Show inbound emails in the  email compose from
     - Show and enable the option "Users may send as themselves"
- Update look and feel of the "from" dropdown

![image](https://user-images.githubusercontent.com/1140849/213172144-0afb4428-22f2-47d0-851a-06861df02ad8.png)
 

## Motivation and Context

- The compose from dropdown options should be taken from outbound email accounts
- Allow configuring group outbound email accounts that can be shared by multiple users

## How To Test This

* Add outbound email accounts
* Go to Email compose
* Check that the outbound email accounts should in the from dropdown
* Try sending emails with the different configured outbounds
* If option to allow user to send using system email is configured, user should also be able to use that.
* Also test using the compose modal on the history subpanel 

## Types of changes


### Final checklist
